examples/get_vouchers.php

```diff
@@ -1,17 +1,26 @@
 <?php
-require __DIR__ . '/../vendor/autoload.php';
 
-$getVouchersRequest = new \AllDigitalRewards\Xoxoday\Catalog\GetVouchersRequest('some-fake-access-token');
+use AllDigitalRewards\Xoxoday\Catalog\GetVouchersRequest;
+use AllDigitalRewards\Xoxoday\Client;
+use Psr\Http\Client\ClientExceptionInterface;
+
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
the autoload require can come after the use statements?   this is new to me.   without the autoload you can't use.   but I trust it works. lol